docs/pages/illuminasetup.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: "Illumina Setup"
+order: 1.5
+categories: ["User Manual"]
+---
+
+To use the Illumina support in Run Scanner, the additional package 
+_runscanner-illumina_ must be built.
+
+You will need a copy of [jsoncpp](https://github.com/open-source-parsers/jsoncpp) and an AutoTools-capable build environment with libstdc++ 5 or later. On Debian/Ubuntu:
+
+    sudo apt-get install pkg-config cmake libjsoncpp-dev autoconf libtool build-essential clang-5.0
+
+On Fedora/Red Hat:
+
+    sudo yum groupinstall "Development Tools"
+    sudo yum install pkgconfig cmake jsoncpp-devel libtool autoconf clang
+
+Pull the Illumina Interop code via:
+
+    git submodule init && git submodule update
```

Comment:
Do you need to `cd runscanner-illumina` before this line?

Reply:
Just tried with a fresh clone of runscanner and this command works appropriately from the project root. These and the installation instructions were mostly copied from the respective readme's